src/main/java/com/dumbdogdiner/stickyapi/bukkit/command/AsyncCommand.java

```diff
@@ -43,7 +43,7 @@
      * @param commandName The name of the command the user will execute
      * @param owner       The plugin that owns this command.
      */
-    public AsyncCommand(String commandName, Plugin owner) {
+    public AsyncCommand(@NotNull String commandName, Plugin owner) {
```

Comment:
For future reference: please keep changes unrelated to the branch somewhere else!